File: beecrowd/1/1011/1011.cs
using System;

class URI {

    static void Main(string[] args) {
        double R;
        R = double.Parse(Console.ReadLine());
        Console.WriteLine("VOLUME = {0}", ((4.0/3) * 3.14159 * Math.Pow(R, 3)).ToString("0.000"));
    }
}

File: beecrowd/1/1011/1011.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

class Main {

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);
        double R = sc.nextDouble();
        System.out.printf("VOLUME = %.3f\n", (4.0 / 3) * 3.14159 * Math.pow(R, 3));
        sc.close();

    }

}
